js/src/components/google-combo-account-card/connected-google-combo-account-card.js

```diff
@@ -93,19 +97,31 @@ const ConnectedGoogleComboAccountCard = () => {
 		return null;
 	};
 
+	const getCardDescription = () => {
+		return (
+			<>
+				<AccountCreationDescription
+					isCreatingWhichAccount={ isCreatingWhichAccount }
+				/>
+			</>
+		);
+	};
+
+	const showSuccessNotice =
+		googleAdsAccount.status === GOOGLE_ADS_ACCOUNT_STATUS.CONNECTED ||
+		googleAdsAccount.step === 'link_merchant';
+
 	return (
 		<AccountCard
 			appearance={ APPEARANCE.GOOGLE }
-			alignIcon="top"
 			className="gla-google-combo-account-card--connected"
-			description={
-				<AccountCreationDescription
-					isCreatingWhichAccount={ isCreatingWhichAccount }
-				/>
-			}
+			description={ getCardDescription() }
 			helper={ getHelper() }
 			indicator={ getIndicator() }
-		/>
+		>
+			{ showSuccessNotice && <ConversionMeasurementNotice /> }
+			<ClaimAdsAccount />
```

Comment:
@ankitrox Check my previous comments. Let's add the condition to display `ClaimAdsAccount`